### DISCUSSION TITLE
CommandBar Button double click

### DISCUSSION THREAD

**Discussion by workgroupengineering** [community/community#64210] [Mobile]
### Select Topic Area

Product Feedback

### Body

When double-tapping the CommandBar button, it scrolls the list to the top.

**Comment by Iron-Ham** [comment]
Hi @workgroupengineering,

Thank you for the feedback! It's user feedback that helps us make sure that GitHub Mobile continuously improves your experience. 
I'd like to request a bit more information about what you're expecting to see and what's not working for you. Could you provide me with the version of the app that you're running, as well as what platform you're on (iOS, Android, and the version of the operating system)? 

Thank you,
Hesham

**Reply by workgroupengineering** [reply]
# Use case **Mobile App** **v.1.126. (739)** os **Android 13**:

List with more item like notifications list or explore list.

**Comment by bronglil** [comment]
@workgroupengineering Double-tapping is a common gesture used in various apps, but it's important to ensure that it doesn't conflict with other actions. You might want to adjust the gesture recognition sensitivity to avoid triggering the double-tap action accidentally and also If the double-tap action is causing confusion or frustration for users, consider changing the functionality altogether. For example, you could use the double-tap action for a different purpose that aligns with the app's functionality and user expectations.

This is an example of the react native, you can use this for better understanding.

`import React, { Component } from 'react';
import { TouchableOpacity, Text } from 'react-native';

class CommandBarButton extends Component {
  constructor(props) {
    super(props);
    this.lastPress = 0; // To track the time of the last press
  }

  handlePress = () => {
    const currentTime = new Date().getTime();
    const doubleTapThreshold = 300; // Adjust this value as needed

    if (currentTime - this.lastPress < doubleTapThreshold) {
      // Double-tap action
      this.props.onDoubleTap();
    }
    
    this.lastPress = currentTime;
  };

  render() {
    return (
      <TouchableOpacity onPress={this.handlePress}>
        <Text>{this.props.buttonText}</Text>
      </TouchableOpacity>
    );
  }
}

export default CommandBarButton;
`

**Comment by workgroupengineering** [comment]
> ### Select Topic Area
> 
> Product Feedback
> 
> ### Body
> 
> When double-tapping the CommandBar button, it scrolls the list to the top.

Fixed in version 1.180.0
